docs/source/blocks/blocks-core/teaser.md

```diff
@@ -0,0 +1,152 @@
+---
+myst:
+  html_meta:
+    "description": "Volto Teaser Block description and developer notes"
+    "property=og:description": "Volto Teaser Block description and developer notes"
+    "property=og:title": "Volto Teaser Block"
+    "keywords": "Volto, React, blocks, teaser, Plone"
+---
+
+(teaser-block)=
+
+# Teaser Block
+
+Volto comes with a Teaser block that given a content object as target is able to pull information from it and show it in the block.
+By default, this information is the `title`, `description`, `preview_image` and shows them in a summary (Image on left, title, description) way.
+
+## Blocks settings
+
+The Teaser has the following settings:
+
+Target
+: The target is either an existing content item in your Plone site that can be selected by clicking the list icon, or an external URL that can be typed into the text field and saved by clicking the right arrow when it appears after you start typing.
+
+Title
+: The title is the title of the content item.
+
+Head title
+: The head title is a heading that appears above the title.
+
+Description
+: The description is plain text that summarizes or describes the content item.
+
+Image override
+: The image override is either an existing image in your Plone site that can be selected by clicking the list icon, or an external URL of an image that can be typed into the text field and saved by clicking the right arrow when it appears after you start typing.
+
+## Style Wrapper Properties
```

Comment:
We use sentence case for headings, with proper nouns in Title Case and acronyms in uppercase.

```suggestion
## Style wrapper properties
```